README.md

```diff
@@ -10,16 +10,25 @@
 
 Retrolambda needs java8 to be installed on your system
 ```
-export ANDROID_HOME=/path/to/android/sdk
+$ export ANDROID_HOME=/path/to/android/sdk
+$ git clone https://github.com/RocketChat/Rocket.Chat.Android.git
+$ cd Rocket.Chat.Android
+$ echo "sdk.dir="$ANDROID_HOME > local.properties
+$ ./gradlew assembleDebug
+(> gradlew assembleDebug on Windows)
+```
 
-git clone https://github.com/RocketChat/Rocket.Chat.Android.git
-cd Rocket.Chat.Android
+### How to send APK to device
 
-echo "sdk.dir="$ANDROID_HOME > local.properties
+The following steps are only needed if running via command line. They are not needed if you are building via Android Studio.
 
-./gradlew assembleDebug
-```
+Ensure that that ADB recognizes your device with `$ adb devices`.
+
+If a single device exists, install via `$ adb install /path/to/apk.apk`.
+
+Assuming you used Gradle like earlier, the file will be called `module_name-debug.apk` in `project_name/module_name/build/outputs/apk/`.
 
+Alternatively, you can simply run `$ ./gradlew installDebug` (`> gradlew installDebug` on Windows)to build, deploy, and debug all in a single command.
```

Comment:
Only a spacing among "[...] Windows)to [...]" and we are go!

Reply:
Done and done.